sequencers/modeshape-sequencer-ddl/src/main/java/org/modeshape/sequencer/ddl/dialect/teiid/TeiidDdlLexicon.java

```diff
@@ -87,6 +87,103 @@
     }
 
     /**
+     * JCR names related to the teiid materialized options.
+     */
+    public interface MaterializedOptions {
+
+        /**
+         * Whether table is materialized
+         */
+        String MATERIALIZED = "MATERIALIZED";
+
+        /**
+         * Materialized table
+         */
+        String MATERIALIZED_TABLE = "MATERIALIZED_TABLE";
+
+        /**
+         * Whether table is updateable
+         */
+        String UPDATABLE = "UPDATABLE";
+
+        /**
+         * Allow Teiid based management
+         */
+        String ALLOW_MATVIEW_MANAGEMENT = "teiid_rel:ALLOW_MATVIEW_MANAGEMENT";
```

Comment:
Then because these string values follow the conventions of JCR names (but are not to be treated as JCR names), does this work correctly? For example, the code might try to parse these into a JCR name. If so, then the `:` should be escaped.